src/CSS/index.css

```diff
@@ -0,0 +1,27 @@
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,300;0,700;1,400&family=Roboto+Slab:wght@300;400;700&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  text-decoration: none;
+}
+
+*,
+*::after,
+*::before {
+  box-sizing: inherit;
+}
+
+html {
+  box-sizing: border-box;
+  font-size: 62.5%;
+  font-family: "Montserrat", sans-serif;
+  scroll-behavior: smooth;
+}
```

Comment:
- [ ] I genuinely appreciate the effort you put into creating a consistent and visually appealing design for your web app. However, as per the design guidelines outlined in [Behance](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), it is recommended to use `Lato` as the designated font for controls, and `Gill Sans` for the body text and titles. Ensuring adherence to these font choices will help maintain the intended aesthetic and cohesion of the design. Keep up the great work 👏